From 018b03344589fba60b8c2ffa172d1815d9025644 Mon Sep 17 00:00:00 2001
From: Liwei Song <liwei.song@windriver.com>
Date: Sun, 30 Aug 2015 17:49:37 +0800
Subject: [PATCH 2/2] enable Display audio driver power well request for
 baytrail

This patch is use to enable Display audio driver in baytail platform,
It use the same power domain method API as Haswell and Broadwell.
Since HDMI codec on Bytrail is also in the shared power well with GPU,
this is use to avoid communication failure with the codec.

Meanwhile, it avoid failure when HDMI/VGA hotplug, because when hotplug
happened, all the other display of domains was disabled, without this
audio display, the communication with codec will fail.

Upstream has fixed this by update i915 audio, hda driver and component API.

Signed-off-by: Liwei Song <liwei.song@windriver.com>
Signed-off-by: chunguang yang <chunguang.yang@windriver.com>
---
 drivers/gpu/drm/i915/intel_pm.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index 6c416c4..8861eec 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -7030,6 +7030,7 @@ int intel_power_domains_init(struct drm_i915_private *dev_priv)
 		set_power_wells(power_domains, chv_power_wells);
 	} else if (IS_VALLEYVIEW(dev_priv->dev)) {
 		set_power_wells(power_domains, vlv_power_wells);
+		hsw_pwr = power_domains;
 	} else {
 		set_power_wells(power_domains, i9xx_always_on_power_well);
 	}
-- 
1.7.5.4

